WeartGloves/CMakeLists.txt

```diff
@@ -0,0 +1,100 @@
+#/*
+# Copyright 2021 Istituto Italiano di Tecnologia (IIT)
+# @author Ehsan Ranjbari <ehsan.ranjbari@iit.it>
+#* CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+#*/
+cmake_minimum_required(VERSION 3.5)
+
+project(YARPWeArtGlove VERSION "0.0.1"
+                  LANGUAGES CXX C)
+
+set(WeArtGloveLibName "WeArtGloveLib")
+set(WeArtGloveDevName "WeArtGlove")
+
+find_package(YARP REQUIRED)                
+find_package(${WeArtGloveLibName} REQUIRED)
+find_package(IWear REQUIRED)
+find_package(Eigen3 REQUIRED)
+find_package(iDynTree REQUIRED)
+find_package(BipedalLocomotionFramework 0.9.0
+  COMPONENTS VectorsCollection IK ParametersHandlerYarpImplementation
+             ContinuousDynamicalSystem ManifConversions
+             ParametersHandlerYarpImplementation REQUIRED)
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+include(GNUInstallDirs)
+
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+
+# Export all symbols in Windows
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+
+set(YARP_FORCE_DYNAMIC_PLUGINS ON)
+
+# Add a postfix to Windows libraries compiled in debug
+if(MSVC)
+    set(CMAKE_DEBUG_POSTFIX "d")
+    
+    #NOMINMAX is to avoid windows.h defining its own versions of min and max
+    add_definitions(-DNOMINMAX)
+endif()
+
+# Libraries type
+if(MSVC)
+    option(BUILD_SHARED_LIBS "Build libraries as shared as opposed to static" OFF)
+else()
+    option(BUILD_SHARED_LIBS "Build libraries as shared as opposed to static" ON)
+endif()
```

Comment:
Can we default to `BUILD_SHARED_LIBS` set to `ON` for both Windows and rest of OS for consistency?